src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Kindly enable the linters so that the errors can be found and fixed. We have to understand that linters play an important role in our projects because it highlights errors in making our code free from unnecessary errors.